component/file_cache/lfu_policy.go

```diff
@@ -153,14 +152,7 @@ func (l *lfuPolicy) clearItemFromCache(path string) {
 	}
 
 	// There are no open handles for this file so its safe to remove this
-	err := deleteFile(path)
-	if err == nil {
-		// File was deleted so try clearing its parent directory
-		dirPath := filepath.Dir(path)
-		if dirPath != l.tmpPath {
-			os.Remove(dirPath)
-		}
-	}
+	deleteFile(path)
```

Comment:
this is related to the fio issue - could we mention in the comments here that we would just have left empty directories that will be cleaned up on `stop()`

Reply:
As Gauri is fixing this in a separate PR, just to be consistent in my test case have removed this code from here.